README.md

```diff
@@ -304,6 +304,40 @@ Exclude any BDD scenario or feature file tagged with `@manual`.
 
 Default: `true`
 
+#### behaveTestEnvironmentVariables ####
+
+Enables the ability to set environment variables when executing the behave tests. The environment variables should be 
+defined in the pom.xml:
+```xml
+<plugin>
+  <groupId>org.technologybrewery.habushu</groupId>
+  <artifactId>habushu-maven-plugin</artifactId>
+  <extensions>true</extensions>
+  <configuration>
+    <behaveTestEnvironmentVariables>
+      <ENV_VAR>VALUE</ENV_VAR>
+    </behaveTestEnvironmentVariables>
+  </configuration>
+</plugin>
+```
```

Comment:
I: Added additional instructions for adding environment variables through the cli because it differs slightly than other configuration options. From my experience with this, if you want to be able to pass environment variables as `-Dhabushu...` without any `pom.xml` configuration, the `behaveTestEnvironmentVariables` parameter would need to be a string. We could then parse the string and convert to a map but entries in the pom (not using the cli) would need to follow the format: 
```
<behaveTestEnvironmentVariables>ENV_VAR1=value1;ENV_VAR2=value2</behaveTestEnvironmentVariables>
```

Since envrionment variables are usually set in the pom and follow the format:
```
<behaveTestEnvironmentVariables>
      <ENV_VAR>VALUE</ENV_VAR>
      <ENV_VAR2>VALUE2</ENV_VAR2>
</behaveTestEnvironmentVariables>
```
I chose to go with `behaveTestEnvironmentVariables` as a map (instead of a string to parse) and you still have the ability to pass in environment variables with `-D` option if you set up the pom correctly.

Issues/comments/concerns here let me know.

Reply:
https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Mapping_Collection_Types

I think you went with the right approach -  this is the standard format for maps in poms and there's no inherent comma-separated string parsing like there is for lists.